test/runtests.sh

```diff
@@ -0,0 +1,4 @@
+set -e
+set -x
+npm run compile
```

Comment:
I would prefer to test the actual output of babel's transpile than the in memory cached copy of babel-node.  I know that they should be identical, but this feels safer to me.  Babel-node still has to do translations, so I don't think we're really saving time by using it.